packages/react/src/components/OverflowMenuItem/OverflowMenuItem.js

```diff
@@ -195,7 +182,10 @@ export default class OverflowMenuItem extends React.Component {
               <TagToUse
                 {...other}
                 {...{
-                  'data-floating-menu-primary-focus': primaryFocus || null,
+                  'data-floating-menu-primary-focus':
+                    (FeatureFlags.enabled('enable-v11-release') &&
+                      other.primaryFocus) ||
+                    null,
```

Comment:
With `primaryFocus` being removed, should this just be removed?

Reply:
@joshblack I wasn't sure if I should remove it completely bc there's no `/next` component for this, as in this file is used for v10 and v11, and it only needs to be removed for v11.

Reply:
@jnm2377 if it helps, you can treat this file at the moment as "v11". So if you want to remove it for v11 go for it 👍  All the files for v10 are under the v10 branch now